primitives/runtime/src/traits.rs

```diff
@@ -540,6 +540,9 @@ morph_types! {
 	/// Morpher to disregard the source value and replace with another.
 	pub type Replace<V: TypedGet> = |_| -> V::Type { V::get() };
 
+	/// Morpher to disregard the source value.
+	pub type Ignore = |_| -> () { () };
```

Comment:
```suggestion
	pub type Ignore = |_| { };
```

Reply:
```
 --> primitives/runtime/src/traits.rs:544:24
    |
405 | macro_rules! morph_types {
    | ------------------------ when calling this macro
...
544 |     pub type Ignore = |_| {};
    |                           ^ no rules expected this token in macro call
    |
note: while trying to match `->`
   --> primitives/runtime/src/traits.rs:471:9
    |
471 |         = |_| -> $outcome:ty { $( $ex:expr )* };
    |               ^^
```

Reply:
I don't know which macro it is, but judging from the syntax, you can also just do the following:

```suggestion
	pub type Ignore = |_| -> () {};
```